src/ILLink.Tasks/LinkTask.cs

```diff
@@ -303,6 +316,13 @@ protected override string GenerateResponseFileCommands ()
 			if (OutputDirectory != null)
 				args.Append ("-out ").AppendLine (Quote (OutputDirectory.ItemSpec));
 
+			if (NoWarn != null)
+				args.Append ($"--nowarn \"{NoWarn}\"");
+
+			if (_trimmerAnalysisWarnings is bool trimmerAnalysisWarnings &&
+				trimmerAnalysisWarnings == true)
+				args.Append ("--nowarn 2006;2026;2037;2043");
```

Comment:
As noted in the parser below, linker should only accept prefixed codes, so `IL2006;IL2026;IL2037;IL2043`.

Reply:
I also find it weird that this set doesn't match the set of warnings in the new TrimCorrectness subcategory. Maybe I don't understand the intent of the subcategory...

Reply:
I used the `TrimCorrectness` subcategory in all warnings related to trimming analysis and the usage of attributes related to it. But I think you're right that warnings about invalid usage of the attributes are not `TrimCorrectness` warnings, so I'm changing this. I'm also dropping the IL2037 from the list of `TrimCorrectness` warnings since I think we should always warn on unresolved members.

With this, the set of warnings we turn off by default is the set of warnings that use the `TrimCorrectness` subcategory.